src/Analysis/GWASV2/Components/Diagrams/CohortsOverlapDiagram/CohortsOverlapDiagram.stories.jsx

```diff
@@ -60,7 +60,7 @@ SuccessCase.parameters = {
             ctx.delay(1100),
             ctx.json({
               cohort_overlap: {
-                overlap_after_filter: Math.floor(Math.random() * 500),
+                case_control_overlap: Math.floor(Math.random() * 500),
```

Comment:
I'm unable to run this storybook story: 

<img width="1432" alt="image" src="https://user-images.githubusercontent.com/113449836/205691857-7ca67a47-7d66-4bc0-bdd6-747db5a939c9.png">